src/test/java/com/Automation/APIAutomation/UserManagement/postUsers.java

```diff
@@ -63,7 +58,7 @@ public void validatePostWithString() {
                 .body("{\"name\":\"pallavi\",\"job\":\"QA\"}")
```

Comment:
use JsonObect to construct this kind of strings

Reply:
Update changes .